docs/tutorials/speech.rst

```diff
@@ -108,78 +89,64 @@ turns out that the algorithm they use is written in Python like this::
         "EXTERMINATE",
     ]
 
-    # Loop over each line in the poem and use the speech module to recite it.
+    # Wprowadź w pętlę każdą linię wiersza i użyj modułu
     for line in poem:
         speech.say(line, speed=120, pitch=100, throat=100, mouth=200)
         sleep(500)
 
-As the comments demonstrate, it's a very simple in design:
-
-* Named fragments (``location``, ``prop``, ``attitude`` etc) are randomly generated from pre-defined lists of possible values. Note the use of ``random.choice`` to select a single item from a list.
-* A template of a poem is defined as a list of stanzas with "holes" in them (denoted by ``{}``) into which the named fragments will be put using the ``format`` method.
-* Finally, Python loops over each item in the list of filled-in poetry stanzas and uses ``speech.say`` with the settings for the DALEK voice to recite the poem. A pause of 500 milliseconds is inserted between each line because even DALEKs need to take a breath.
-
-Interestingly the original poetry related routines were written by Davros in
-`FORTRAN <https://en.wikipedia.org/wiki/Fortran>`_ (an appropriate
-language for DALEKS since you type it ALL IN CAPITAL LETTERS). However, The
-Doctor went back in time to precisely the point between Davros's
-`unit tests <https://en.wikipedia.org/wiki/Unit_testing>`_
-passing and the
-`deployment pipeline <https://en.wikipedia.org/wiki/Continuous_delivery>`_
-kicking in. At this instant he was able to insert a MicroPython interpreter
-into the DALEK operating system and the code you see above into the DALEK
-memory banks as a sort of long hidden Time-Lord
-`Easter Egg <https://en.wikipedia.org/wiki/Easter_egg_(media)>`_ or
+Jak pokazują komentarze, jest to bardzo prosty model:
+
+* Nazwane fragmenty (``location``, ``prop``, ``attitude`` itd.) są generowane losowo z predefiniowanych list możliwych wartości. Zwróć uwagę na użycie ``random.choice`` w celu wybrania pojedynczego elementu z listy.
+* Szablon wiersza definiowany jest jako lista zwrotek z "dziurami" (oznaczonymi przez ``{}``), do których za pomocą metody ``format`` wstawione zostaną nazwane fragmenty.
+* Na koniec, Python wykonuje pętlę nad każdym elementem na liście wypełniaczy poezji i używa ``speech.say`` z ustawieniami głosu DALEKa do recytowania wiersza. Między poszczególnymi liniami wstawiana jest pauza 500 milisekund, ponieważ nawet DALEKowie muszą odetchnąć.
+
+Co ciekawe, oryginalne wytyczne związane z poezją zostały napisane przez
+Davrosa w języku `FORTRAN <https://en.wikipedia.org/wiki/Fortran>`_
+(odpowiedni język DALEKów, gdyż pisany jest TYLKO WIELKIMI LITERAMI). Jednakże
+doktor cofnął się w czasie dokładnie do punktu pomiędzy wprowadzaniem
+`testów jednostkowych <https://pl.wikipedia.org/wiki/Test_jednostkowy>`_ Davros'a i
+`potków wdrożeniowych <https://en.wikipedia.org/wiki/Continuous_delivery>`_ .
```

Comment:
Co to są "potki"? To jakiś polski slang informatyczny? W elektronice to potencjometry, ale nie spotkałem się w informatyce.